R/ReferenceFrameRotations/Compat.toml

```diff
@@ -7,11 +7,14 @@ StaticArrays = "0.0-0.10"
 ["0.3-0.4"]
 StaticArrays = "0.8.3-0.10"
 
-["0.3-0.5"]
+["0.3-0.5.1"]
 julia = "0.7-1.3"
 
 ["0.5"]
 StaticArrays = "0.9.2-0.10"
 
-["0.5.1"]
+["0.5.1-0.5.2"]
 Crayons = "1.0-4.0"
+
+["0.5.2"]
+julia = "1.0-1.1"
```

Comment:
This is wrong, see https://github.com/JuliaComputing/Registrator.jl/issues/103. Linking it with that issue so we can correct it later.